cfpb/consumerfinance.gov
# TCCP: Intersperse speed bump wells into list of card results

Adds situation-specific "speed bumps" after every fifth credit card on the results page. Also cleans up some unrelated CSS.

I didn't remove the current situation banners at the top of the screen. We'll need to morph those into the `Cards you’re looking for have these features` bulleted list in a future PR.

@chosak I left all the content in the single jinja template but I'm happy to rework it when we remove the situation banners.

@caheberer It doesn't perfectly match figma because I used our default [well pattern](https://cfpb.github.io/design-system/patterns/wells) (without a heading). If it tests well I can further tweak the design.

## How to test this PR

1. `./frontend.sh`
1. Visit the [cards page](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/?credit_tier=Credit%20scores%20from%20620%20to%20719&location=AL&rewards=Cashback%20rewards&rewards=Travel-related%20rewards&rewards=Other%20rewards&situations=Make%20a%20big%20purchase&situations=Build%20credit&ordering=product_name) with a few situations in the URL and you should see a speed bump message after every fifth result.
1. `yarn cypress run --spec test/cypress/integration/consumer-tools/credit-cards/explore-cards.cy.js`


## Screenshots

<img width="343" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/1143839e-6fc0-4b97-8d0d-f7eb9ab20be2">


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
